image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -172,7 +172,13 @@ spec:
         - name: {{ $extraMount.name }}
           {{- $extraMount.mount | toYaml | nindent 10 }}
         {{- end }}
-        {{- range $mount := (default list ._rox.central.declarativeConfiguration.mounts) }}
+        {{- range $mount := (default list ._rox.central.declarativeConfiguration.mounts.configMaps) }}
+        - name: {{ $mount }}
+          mountPath: /run/secrets/stackrox.io/declarative-configuration/
```

Comment:
I'm just now realising that using `/run/secrets/...` path is not entirely correct considering that most of declarative configuration has nothing to do with secrets

Reply:
Yeah, its unfortunate that neither `config` nor `secrets` really matches and we probably have to use the non-default path for mounts in central.

I could switch it to `/run/stackrox.io/declartive-config` instead. WDYT?